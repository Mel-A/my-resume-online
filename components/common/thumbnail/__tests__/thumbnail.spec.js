import { shallow } from 'enzyme'
import Thumbnail from '..'

test('thumbnail renders correctly', () => {
  const thumbnail = shallow(<Thumbnail />)
  expect(thumbnail).toMatchSnapshot()
});