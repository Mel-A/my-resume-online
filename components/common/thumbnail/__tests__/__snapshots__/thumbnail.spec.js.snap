// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thumbnail thumbnail renders correctly 1`] = `ShallowWrapper {}`;

exports[`Thumbnail thumbnail renders theme correctly 1`] = `ReactWrapper {}`;
